examples/tomography_state.ipynb

```diff
@@ -5,74 +5,390 @@
+    "\n",
+    "$$ |\\Psi\\rangle = \\frac{1}{2}\\begin{pmatrix} 1\\\\ 1 \\\\ 1\\\\ -1\\end{pmatrix}= {\\rm CZ}(0,1){\\rm H}(1){\\rm H}(0)|0,0\\rangle,$$\n",
+    "\n",
+    "which corresponds to the state matrix\n",
+    "\n",
+    "$$ \\rho_{\\rm true} = |\\Psi\\rangle\\langle \\Psi| = \\frac{1}{4}\\begin{pmatrix} 1 & 1 & 1 &-1\\\\ 1 & 1 & 1 &-1 \\\\ 1 & 1 & 1 &-1\\\\ -1 & -1 & -1 & 1\\end{pmatrix}.$$"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
     "import numpy as np\n",
-    "from pyquil import Program, get_qc\n",
+    "from pyquil import Program\n",
     "from pyquil.gates import *"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": 3,
    "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([[ 0.25,  0.25,  0.25, -0.25],\n",
+       "       [ 0.25,  0.25,  0.25, -0.25],\n",
+       "       [ 0.25,  0.25,  0.25, -0.25],\n",
+       "       [-0.25, -0.25, -0.25,  0.25]])"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "## Construct a state with a `Program`\n",
-    "We'll construct a two-qubit graph state by Hadamarding all qubits and then applying a controlled-Z operation across edges of our graph. In the two-qubit case, there's only one edge. "
+    "# numerical representation of the true state\n",
+    "Psi = (1/2) * np.array([1, 1, 1, -1])\n",
+    "rho_true = np.outer(Psi, Psi.T.conj())\n",
+    "\n",
+    "rho_true"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "H 0\n",
+      "H 1\n",
+      "CZ 0 1\n",
+      "\n"
+     ]
+    }
+   ],
    "source": [
+    "# construct the state preparation program\n",
+    "\n",
     "qubits = [0, 1]\n",
-    "program = Program()\n",
+    "state_prep_prog = Program()\n",
+    "\n",
     "for qubit in qubits:\n",
-    "    program += H(qubit)\n",
-    "program += CZ(qubits[0], qubits[1])\n",
-    "print(program)"
+    "    state_prep_prog += H(qubit)\n",
+    "    \n",
+    "state_prep_prog += CZ(qubits[0], qubits[1])\n",
+    "\n",
+    "print(state_prep_prog)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Construct a `ObservablesExperiment` for state tomography\n",
-    "We can print this out to see the 16 measurements we will perform."
+    "## Step 2. Construct a `ObservablesExperiment` for state tomography\n",
+    "\n",
+    "We use the helper function `generate_state_tomography_experiment` to construct a tomographically complete set of measurements.\n",
+    "\n",
+    "We can print this out to see the 16 observables or operator measurements we will perform."
```

Comment:
```suggestion
    "We can print this out to see the 15 observables or operator measurements we will perform. Note that we could have included an additional observable `I0I1`, but since this trivially gives an expectation of 1 we instead omit this observable in experiment generation and include its contribution by hand in the estimation methods. Be mindful of this if generating your own settings."
```